### PR TITLE
Docs: fix redirected links to user documents

### PR DESCRIPTION
**Pull request by t-hamano**
Follow-up on: #49533
Related core ticket: [#57726 (Use the new /documentation/ URLs for HelpHub links) – WordPress Trac](https://core.trac.wordpress.org/ticket/57726)

## What?
This PR fixes a problem with the redirection of links to documentation for users.

## Why?

Just like the changes made in the core, I think all `/support/article/*` URLs should be redirected to `/documentation/article/` to avoid an extra redirect.

## How?

- Search Gutenberg projects with `https://wordpress.org/support/`.
- Open the found link in a browser.
- Updated the article that is being redirected.

## Testing Instructions

- Check changed files.
- Access the URL before the change.
- Confirm that you are redirected to the changed URL.

### CODE REVIEW

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ Welcome to Gutenberg, a WordPress project. We hope you join us in creating the f
 
 * Join us on Slack for real-time communication, it is where maintainers coordinate around the project. To get started using Slack, see: https://make.wordpress.org/chat/
 
-* For general WordPress support with the core editor, see the [WordPress.org support forums](https://wordpress.org/support/) — it is highly active and well maintained.
+* For general WordPress support with the core editor, see the [WordPress.org support forums](https://wordpress.org/support/forums/) — it is highly active and well maintained.
```
Comment (t-hamano) [inline .github/SUPPORT.md:12]: This change is the only exception. This URL should be in the support forum, but accessing `https://wordpress.org/support/` redirects to `https://wordpress.org/documentation/`. Therefore, we have rewritten it to the appropriate link.

File: packages/edit-site/src/components/welcome-guide/styles.js [hunk 2]
```diff
@@ -119,7 +119,7 @@ export default function WelcomeGuideStyles() {
 								) }
 								<ExternalLink
 									href={ __(
-										'https://wordpress.org/support/article/styles-overview/'
+										'https://wordpress.org/documentation/article/block-based-widgets-editor/'
```
Comment (alexstine) [inline packages/edit-site/src/components/welcome-guide/styles.js]: Looks like wrong link.

```suggestion
										'https://wordpress.org/documentation/article/styles-overview/'
```

**Review by alexstine — COMMENTED** [review]
Looks like one needed fix and then this should be good. 👇

File: packages/edit-site/src/components/welcome-guide/styles.js (hunk 2 above)
Comment (t-hamano) [reply packages/edit-site/src/components/welcome-guide/styles.js]: Oops, thanks for finding the mistake!

**Review by alexstine — APPROVED** [review]
I think this is good to go but may not be a bad idea to wait on a 2nd review. I would assume this new docs area is fully ready but honestly do not know for certain.

**Pull request merged**